package.json

```diff
@@ -37,16 +37,23 @@
         "@aws-cdk/aws-eks": "1.119.0",
         "@aws-cdk/aws-elasticsearch": "1.119.0",
         "@aws-cdk/aws-iam": "1.119.0",
+        "@aws-cdk/aws-kms": "1.119.0",
         "@aws-cdk/aws-logs": "1.119.0",
+        "@aws-cdk/aws-s3": "1.119.0",
         "@aws-cdk/aws-secretsmanager": "1.119.0",
         "@aws-cdk/core": "1.119.0",
         "@aws-cdk/pipelines": "1.119.0",
         "@types/bcrypt": "^5.0.0",
         "aws-sdk": "^2.952.0",
         "bcrypt": "^5.0.1",
+        "deepmerge": "^4.2.2",
         "js-yaml": "3.13.1",
+        "lint": "^0.7.0",
         "sync-request": "6.1.0",
         "user": "^0.0.0",
         "yaml": "^1.10.2"
-    }
+    },
+    "bundledDependencies": [
```

Comment:
Do we need this?  E.g. do you not expect this to come from npm? (maybe a moot point if we decide to move forward with ts-deepmerge).

Reply:
yes resolved with the suggestion.